test/python/pulse/test_builder.py

```diff
@@ -811,31 +811,6 @@ def get_sched(qubit_idx: [int], backend):
 class TestSubroutineCall(TestBuilder):
     """Test for calling subroutine."""
 
-    def test_call(self):
-        """Test calling schedule instruction."""
-        d0 = pulse.DriveChannel(0)
-        d1 = pulse.DriveChannel(1)
-
-        reference = pulse.Schedule()
-        reference = reference.insert(10, instructions.Delay(10, d0))
-        reference += instructions.Delay(20, d1)
-
-        ref_sched = pulse.Schedule()
-        with self.assertWarns(DeprecationWarning):
-            ref_sched += pulse.instructions.Call(reference)
```

Comment:
I think you can just add `reference` directly here instead of casting it into Call. This test is against builder call command, so should not be removed.